lib/screens/e_commerce/order_detail.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:promdi_fe/screens/e_commerce/eWidgets/order_status.dart';
+
+class OrderDetail extends StatefulWidget {
+  const OrderDetail({Key? key}) : super(key: key);
+
+  @override
+  _OrderDetailState createState() => _OrderDetailState();
+}
+
+class _OrderDetailState extends State<OrderDetail> {
```

Comment:
My assumption is that we shall be using the actual order details here, instead of the current dummy data.